### PR TITLE
Minor adjustment to angle translation

### PR DESCRIPTION
**Pull request by MikeGitb**
Makes the conversion of angle from wire format to millidegree imho more readable.

### CODE REVIEW

**Comment by MikeGitb** [comment]
Sorry, got mixed up in my own branch mess

File: libsweep/src/sweep.cc [hunk 1]
```diff
@@ -218,7 +218,7 @@ void sweep_device_accumulate_scans(sweep_device_s device) try {
       out->count = received - 1; // minus 1 to exclude sync reading
       for (int32_t it = 0; it < received - 1; ++it) {
         // Convert angle: compact serial format -> float (in degrees) -> int (in milli-degrees)
```
Comment (dcyoung) [inline libsweep/src/sweep.cc]: Not a big issue, but might remove these comments too now.

Comment (MikeGitb) [inline libsweep/src/sweep.cc]: Done

**Comment by dcyoung** [comment]
Woops, forget to indicate both comments. The same comment on line 478 in `get_scan_direct()` could be removed. Then I'll get this merged!

**Comment by dcyoung** [comment]
I went ahead and removed the extra comment and merged. Thanks MikeGitb

**Comment by MikeGitb** [comment]
Thanks

**Pull request merged**